docs/http/traffic-policy/index.mdx

```diff
@@ -60,8 +60,8 @@ and actions that should take effect.
 
 ## Behavior
 
-Policy rules are evaluated sequentially in the order they are configured with inbound rules taking effect before the
-upstream server is reached and outbound rules taking effect after the upstream server responds. Whether or not the configured
+Policy rules are evaluated sequentially in the order they are configured with `on_http_request` rules taking effect before the
+upstream server is reached and `on_http_response` rules taking effect after the upstream server responds. Whether or not the configured
 actions are performed is determined at runtime by the expressions.
 
```

Comment:
Need `on_tcp_connect` as well. Also in these files seems we may want to be calling Policy 'Traffic Policy' instead?